src/test/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinderTest.java

```diff
@@ -129,6 +130,9 @@ public static void assertRevisionAction(WorkflowRun build) {
         }
     }
 
+    static {
+        System.setProperty(SubversionSCMFileSystem.DISABLE_PROPERTY, "true");
+    }
```

Comment:
Do we need to tear this down in between tests, or do we always want it to be `true`, so it doesn't really matter?

Reply:
Should not really matter I think.